pkg/deploy/api/types.go

```diff
@@ -441,6 +441,20 @@ type DeploymentConfigRollbackSpec struct {
 	IncludeStrategy bool
 }
 
+// DeploymentRequest is a request to a deployment config for a new deployment.
+type DeploymentRequest struct {
+	unversioned.TypeMeta
+	// Name of the deployment config for requesting a new deployment.
+	Name string
+	// DryRun if true should not persist the updated deployment config in storage.
+	DryRun bool
+	// Latest will update the deployment config with the latest images found in the cluster.
+	Latest bool
```

Comment:
Then change the godoc to state that it will apply the latest state from all triggers, including images.  My concern here is that we can't actually implement this in a composable way (we have no way to let arbitrary controllers participate), but that's ok.

If a controller is paused and I call this, what happens?